glean_parser/util.py

```diff
@@ -420,22 +420,24 @@ def remove_output_params(d, output_params):
 
 
 # Names of parameters that only apply to some of the metrics types.
+# **CAUTION**: This list needs to be in the order the Swift & Rust type constructors
+# expects them. (The other language bindings don't care about the order).
 extra_metric_args = [
     "time_unit",
     "memory_unit",
     "allowed_extra_keys",
     "reason_codes",
-    "bucket_count",
-    "range_max",
     "range_min",
+    "range_max",
```

Comment:
Uh oh. Does this mean it was broken before?

Reply:
Well, only for Rust and for that it wasn't supported so far.

Reply:
(and technically for Swift, but it has no implementation for it either)